frontend/ui/widget/multiconfirmbox.lua

```diff
@@ -126,6 +153,14 @@ function MultiConfirmBox:onClose()
     return true
 end
 
+function MultiConfirmBox:onTapClose(arg, ges)
+    if ges.pos:notIntersectWith(self[1][1].dimen) then
```

Comment:
For maintainability it's better to use named references in cases like this.

Reply:
Cut & pasted from sister module confirmbox, and there are quite a few `self[1][1].dimen` already there in both files. (I'd rather not change them, to not add some noise to this PR purpose.)